src/modules/AuthModule/components/Login/Login.tsx

```diff
@@ -1,7 +1,123 @@
+       });
+    }
+  }
+
   return (
-    <div>Login</div>
-  )
+    <Box>
+      <Grid container className="login-grid" spacing={0}>
+        <Grid item xs={5}>
+            <Typography variant="h5" color="black" sx={{ marginTop: "15px" }}>
+              <Typography
+                variant="h5"
+                sx={{ display: "inline-block" }}
+                color="rgba(50, 82, 223, 1)"
+              >
+                Stay
+              </Typography>
+              cation.
+            </Typography>
+            <Grid xs={9} sx={{ marginX: "auto", marginY: "35px" }}>
+              <Typography
+                variant="h5"
+                sx={{ fontWeight: "bold", marginY: "20px" }}
+                color="initial"
+              >
+                Sign in
+              </Typography>
+              <Typography variant="body1" color="initial">
+                If you don’t have an account register
+              </Typography>
+              <Typography variant="body1" color="initial">
+                You Can <Link to={"/register"}>Register Here !</Link>{" "}
+              </Typography>
+            </Grid>
+            <Grid xs={9} sx={{ marginX: "auto" }}>
+              <Box component='form' onSubmit={handleSubmit(onSubmit)}  >
+                <TextField
+                  type="text"
+                  sx={{
+                    marginY: 2,
+                    width: "100%",
+                    backgroundColor: "whitesmoke",
+                  }}
+                  label="Email Address"
+                  placeholder="Please Type Here"
+                 {...register('email' , {required:validateEmail.required ,   pattern: {
+                          value: validateEmail.pattern.value,
+                          message: validateEmail.pattern.message,
+                        }})}
+              />
+              {errors.email && <Alert icon={<Error fontSize="inherit" />} severity="error">
+                 {errors.email.message}
+                 </Alert>}
+                <TextField
+                  type="password"
+                  sx={{
+                    marginY: 2,
+                    width: "100%",
+                    backgroundColor: "whitesmoke",
+                  }}
+                  label="Password"
+                  placeholder="Please Type Here"
+                  {...register('password' , {required: validatePassword.required , pattern: {
+                        value: validatePassword.pattern.value,
+                        message:validatePassword.pattern.message
+                  }})}
+              />
+               {errors.password && <Alert icon={<Error fontSize="inherit" />} severity="error">
+                 {errors.password.message}
+                 </Alert>}
+                <Link className="forget-link" to={'/forget-pass'}>Forgot Password ?</Link>
+                <Button variant="contained" type="submit" sx={{marginY:'25px' , width:'100%'}} disabled={!isDirty || !isValid}>Login</Button>

```

Comment:
We shouldn't use fixed  spacing values. In the MUI'spacing system 1=>8px. We can change marginY:"25px" to marginY:3